contracts/minter-suite/Minters/MinterSEAV0.sol

```diff
@@ -0,0 +1,987 @@
+    {
+        ProjectConfig storage _projectConfig = projectConfig[_projectId];
+        maxInvocations = _projectConfig.maxInvocations;
+        timestampStart = _projectConfig.timestampStart;
+        auctionDurationSeconds = _projectConfig.auctionDurationSeconds;
+        basePrice = _projectConfig.basePrice;
+        auction = _projectConfig.activeAuction;
+    }
+
+    /**
+     * @notice projectId => active auction details.
+     * @dev reverts if no auction exists for the project.
+     */
+    function projectActiveAuctionDetails(
+        uint256 _projectId
+    ) external view returns (Auction memory auction) {
+        ProjectConfig storage _projectConfig = projectConfig[_projectId];
+        auction = _projectConfig.activeAuction;
+        // do not return uninitialized auctions (i.e. auctions that do not
+        // exist, and therefore are simply the default struct)
+        require(auction.initialized, "No auction exists on project");
+        return auction;
+    }
+
+    /**
+     * @notice Convenience function that returns either the current token ID
+     * being auctioned, or the next expected token ID to be auction if no
+     * auction is currently initialized or if the current auction has concluded
+     * (block.timestamp > auction.endTime).
+     * This is intended to be useful for frontends or scripts that intend to
+     * call `createBid` or `settleAndCreateBid`, which requires
+     * the target token ID to be passed in as an argument.
+     * Note that this is not a guarantee that the next token ID to be auctioned
+     * due to edge cases where other minters are being used, but those are not
+     * expected to be encountered in practice.
+     * The function reverts if an invalid project ID is queried, or if the
+     * project has reached its maximum invocations and there is no active
+     * auction.
+     * @param _projectId The project ID
+     * @return The current token ID being auctioned, or the next expected token
+     * ID to be auctioned if no auction is currently initialized or if the
+     * current auction has concluded (block.timestamp > auction.endTime).
+     */
+    function getTokenToBid(uint256 _projectId) external view returns (uint256) {
+        ProjectConfig storage _projectConfig = projectConfig[_projectId];
+        Auction storage _auction = _projectConfig.activeAuction;
+        // if project has an active token auction that is not settled, return
+        // that token ID
+        if (_auction.initialized && (_auction.endTime > block.timestamp)) {
+            return _auction.tokenId;
+        }
+        // otherwise, return the next expected token ID based on core contract
+        // invocation count
+        (
+            uint256 invocations,
+            uint256 maxInvocations,
+            ,
+            ,
+            ,
+
+        ) = genArtCoreContract_Base.projectStateData(_projectId);
+        // revert if next invocation would exceed max invocations
+        require(
```

Comment:
would this be ignoring a manually set max project invocations limit?

Reply:
good call - added additional check for minter max invocations in 8684a99 🙏